Utilities/Distribution/manylinux/imagefiles/cmd.sh

```diff
@@ -25,10 +25,14 @@ mkdir -p ${PYTHONUSERBASE}
 
 function build_simpleitk {
 
-    echo "SIMPLEITK_GIT_TAG: ${SIMPLEITK_GIT_TAG}"
```

Comment:
Is removing the logging message intentional? No need for it, but I assume it was previously useful.

Reply:
With the below feature implemented of using the mounted volume as the source, and taking priority over this variable, it was not worth the effort/logic to correctly report the tag.